manim/mobject/graph.py

```diff
@@ -382,6 +382,34 @@ def construct(self):
                 graph = Graph(list(G.nodes), list(G.edges), layout="partite", partitions=[[0, 1]])
                 self.play(Create(graph))
 
+    The representation of a linear artificial neural network is facilitated
+    by the use of the partite layout and defining partitions for each layer.
+
+    .. manim:: LinearNN
+        :save_last_frame:
+
+        class LinearNN(Scene):
+
+            def construct(self):
+                edges = []
+                partitions = []
+                c=0
+                layers = [2,3,3,2]         # the number of neurons on each layer
+
+                for i in layers:
+                    partitions.append([j for j in range(c+1, c+i+1)])
+                    c += i
+                for i, v in enumerate(layers[1:]):
+                        last = int(np.sum(layers[:i+1]))
```

Comment:
I don't think it is necessary to use `np.sum`, Python's default `sum` does the job. :-)

```suggestion
                        last = sum(layers[:i+1])
```